src/Login.tsx

```diff
@@ -63,6 +90,11 @@ function Login() {
             </ul>
           </div>
         )}
+        {error && (
+          <div className='error-login'>
+            <p>{error.message}</p>
+          </div>
+        )}
```

Comment:
geralmente não é muito usar o operador `{var ?? (<jsx/>)}` com dados que não são booleans, pode causar alguns comportamentos estranhos (como aparecer o erro na tela 2x), uma maneira mais seguro é utilizar ternário
```suggestion
        {error ? (
          <div className='error-login'>
            <p>{error.message}</p>
          </div>
        ) : null}
```